02-microarray/pathway-analysis_microarray_02_gsea.Rmd

```diff
@@ -442,12 +478,12 @@ Let's take a look at 2 different pathways -- one with a highly positive NES and
 
 ### Most Positive NES
 
-Let's look for the gene set with the most positive NES.
+Let's look at the 3 gene sets with the most positive NES.
 
-```{r}
+```{r rownames.print = FALSE}
 gsea_result_df %>%
-  # Use `top_n()` to return the top n observation using `NES` as the ordering variable
-  dplyr::top_n(n = 3, wt = NES)
+  # This orders the results by the NES value and takes the top 3 rows
+  dplyr::slice_max(n = 3, order_by = NES)
```

Comment:
In this spirit of replacing "top" with "max"...
```suggestion
  # This returns the 3 rows with the largest NES values
  dplyr::slice_max(n = 3, order_by = NES)
```

Reply:
Fair enough!